addons/hr_expense/models/hr_expense.py

```diff
@@ -312,8 +312,8 @@ def _get_account_move_line_values(self):
                 'currency_id': expense.currency_id.id if different_currency else False,
             }
             move_line_values.append(move_line_src)
-            total_amount -= move_line_src['debit']
-            total_amount_currency -= move_line_src['amount_currency'] or move_line_src['debit']
+            total_amount += -move_line_src['debit'] or move_line_src['credit']
+            total_amount_currency += -move_line_src['amount_currency'] or -move_line_src['debit'] or move_line_src['credit']
```

Comment:
No related to this fix but this line could be buggy with very low amount in multy-currency. E.g. debit = 0.01 and amount_currency = 0.00. So could you change this line by something like:
```
total_amount_currency += -move_line_src['amount_currency'] if move_line_src['currency_id'] else (-move_line_src['debit'] or move_line_src['credit'])
```